src/index.ts

```diff
@@ -75,12 +75,14 @@ export class Bundler {
   constructor(config: Config, public options: Options = {}) {
     logger.setOptions({ logLevel: options.logLevel })
 
-    this.rootDir = path.resolve(options.rootDir || '.')
+    const rootDir = options.rootDir || '.'
+    this.rootDir = path.resolve(rootDir)
```

Comment:
I guess we don't need to change these lines

Reply:
We don't indeed.

<question>
Could we have a way to support splitting configuration from there for monorepo setup?
Let's say we have settings common for all in './bili.config.js'. 

When we're asking to run bili from another rootDir, we could merge the two bili.config.js (with other possible file names)

From there, we could have a way to check if the value is '.' and we want a way to use bili.config.js from there, then, resolve from ./${rootDir}/bili.config.js and merge values from there.

What do you think?
</question>

Reply:
I prefer the way `tsconfig.json` works:

- You can populate a common `bili.config.js` in root directory, and another `bili.config.js` with `module.exports = {extends:'../../bili.config.js'}` in a leaf package like `packages/foo`
- Then you can run `bili` in `packages/foo`

(not implemented yet)

Reply:
Which part is not implemented yet.
The "extends" key part that'd take the string, resolve its path and (?).

If you want a reproduction  repo with modules in different formats, along with testing you can use the "reproduction repo" in #219 (I can add more use-cases too: tsx,jsx,vue,ts,js,wasm,yaml, -- things rollup plugins can handle-- etc).